pkg/file/file.go

```diff
@@ -27,6 +29,29 @@ func Generate(name string) error {
 		err := createFolder(folder)
 		if err != nil {
 			return err
+		}	
+		
+		//create file for folders other than current dir,main folder and widget folder
+		if(folder != name && folder != data && folder != widget ){
```

Comment:
File creation logic can be moved into another function.

Reply:
@AliAkberAakash  moved the file creation logic to a separate function in the latest commit.